libs/clj-icat-direct/src/clj_icat_direct/queries.clj

```diff
@@ -517,21 +517,19 @@
 
 (def queries
   {:count-all-items-under-folder
-   "WITH user_groups AS ( SELECT g.*
+   "WITH user_groups AS ( SELECT g.group_user_id
                             FROM r_user_main u
                             JOIN r_user_group g ON g.user_id = u.user_id
                            WHERE u.user_name = ?
                              AND u.zone_name = ? ),
 
-         parent      AS ( SELECT * from r_coll_main
+         parent      AS ( SELECT DISTINCT coll_id, coll_name from r_coll_main
```

Comment:
If it is, it's not enforced:

```
ICAT=# \d r_coll_main
                         Table "public.r_coll_main"
      Column      |          Type           |           Modifiers            
------------------+-------------------------+--------------------------------
 coll_id          | bigint                  | not null
 parent_coll_name | character varying(2700) | not null
 coll_name        | character varying(2700) | not null
 coll_owner_name  | character varying(250)  | not null
 coll_owner_zone  | character varying(250)  | not null
 coll_map_id      | bigint                  | default 0
 coll_inheritance | character varying(1000) | 
 coll_type        | character varying(250)  | default '0'::character varying
 coll_info1       | character varying(2700) | default '0'::character varying
 coll_info2       | character varying(2700) | default '0'::character varying
 coll_expiry_ts   | character varying(32)   | 
 r_comment        | character varying(1000) | 
 create_ts        | character varying(32)   | 
 modify_ts        | character varying(32)   | 
Indexes:
    "idx_coll_main2" UNIQUE, btree (parent_coll_name, coll_name)
    "idx_coll_main3" UNIQUE, btree (coll_name)
    "idx_coll_main1" btree (coll_id)
    "idx_coll_main_coll_type" btree (coll_type)
    "idx_coll_main_parent_coll_name" btree (parent_coll_name)

```

Only `coll_name` and `parent_coll_name, coll_name` are made unique.